_assets/js/utils/setOffsetParam.js

```diff
@@ -0,0 +1,6 @@
+// Get the url params offset value and convert it to a string. This is used to set the href link for each pagination button.
+const setOffsetParam = (offsetValue) => {
```

Comment:
Maybe the utils go in one file?  What do you think?

Reply:
Honestly I think it could go either way. I think there are some functions that are conceptually similar and would make sense as a single file, especially the ones that relate to offset, but for the createRange function and other one-off functions, idk maybe it makes sense to leave them as separate files? I'm not sure what would make the most sense to someone totally new to the codebase.